src/reaction.rs

```diff
@@ -405,13 +431,19 @@ Content-Disposition: reaction\n\
         let chat_alice = alice.create_chat(&bob).await;
         let alice_msg = alice.send_text(chat_alice.id, "Hi!").await;
         let bob_msg = bob.recv_msg(&alice_msg).await;
+        assert_eq!(get_chat_msgs(&alice, chat_alice.id, 0).await?.len(), 1);
+        assert_eq!(get_chat_msgs(&bob, bob_msg.chat_id, 0).await?.len(), 1);
 
         bob_msg.chat_id.accept(&bob).await?;
 
         send_reaction(&bob, bob_msg.id, "👍").await.unwrap();
+        expect_reactions_changed_event(&bob, bob_msg.chat_id, bob_msg.id, ContactId::SELF).await?;
+        assert_eq!(get_chat_msgs(&bob, bob_msg.chat_id, 0).await?.len(), 1);
```

Comment:
this test is currently failing, reaction messages are not hidden on the sender's device, see https://github.com/deltachat/deltachat-core-rust/pull/3644#discussion_r998117795